protostar/commands/test/test_command_cache.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+from typing import List
+
+from protostar.testing import TestingSummary
+from protostar.testing.test_results import (
+    FailedTestCaseResult,
+    BrokenTestCaseResult,
+    BrokenTestSuiteResult,
+)
+
+from protostar.self.cache_io import CacheIO
+
+
+class TestCommandCache:
+    def __init__(self, cache_path: Path):
+        self.cache_io = CacheIO(cache_path)
```

Comment:
[In TestCommandCache receive CacheIO instead of Path](https://github.com/software-mansion/protostar/pull/895/commits/a3cf1e5e0463821312e334cfad1c9bcb33c0b62e)